_posts/platform/app/2000-01-01-log-drain.md

```diff
@@ -32,24 +33,101 @@ Our system is also able to send logs to the following SaaS log processing provid
 
 ## Configuration of your Application on Scalingo
 
-As this feature is private beta, you cannot activate logs drain by yourself. You need to add an
-environment variable containing the token to the used logs management service, and contact the
-support so that we activate the logs drain. In a matter of minutes, your logs flow to the logs
-management service.
+The log drains configuration will be done using the [CLI
+](https://doc.scalingo.com/cli).
+
+### Add a Log Drain
+
+To add a log drain management service you will have to use the `log-drains-add`
+command.
+Below, the list of available parameters:
+* `type`: Communication protocol
```

Comment:
I don't find this list of parameter useful. Let's say I'm a user of the platform and I want to add a log drain on my application. Do I care about the `url` flag if I want to use Datadog?

Maybe it would be better to add a link to the correct example in the list of providers of the section "Third-party Provider integrations"? And instead of displaying the various examples of adding a log drain as a list, each provider is a section you can link to. What do you think?